src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableArray`1+Builder.cs

```diff
@@ -49,7 +49,41 @@ internal Builder()
             }
 
             /// <summary>
-            /// Gets or sets the length of the array.
+            /// Get and sets the length of the internal array.  When set the internal array is 
+            /// is reallocated to the given capacity if it is not already the specified length.
+            /// </summary>
+            public int Capacity
+            {
+                get { return _elements.Length; }
+                set
```

Comment:
@sharwell true.

This particular implementation was basically taken from `List<T>::set_Capacity`.  Given it's high use within .Net I error'd in just taking that implementation over devising my own.